.vscode/settings.json

```diff
@@ -20,6 +20,12 @@
     "*.jsx": "javascriptreact"
   },
 
+  // Apply fix all auto-fixable eslint problems on save
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
```

Comment:
> This is an old legacy setting and should in normal cases not be necessary anymore.

[oops](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint#:~:text=vue%22%2C%20%22markdown%22%5D.-,eslint.validate,-%2D%20an%20array%20of)

I'll drop this.